### ISSUE TITLE
Ton could not start

### ISSUE THREAD

**Issue by kindman4** [Lolliedieb/lolMiner-releases#1576]
Both1.48 & 1.49
mine_ton.bat:
set "TONPOOL=stratum+ssl://xxxx:xxxx"
set "TONWALLET=xxxxx.xxxxx"
							
set "EXTRAPARAMETERS=--apiport 1111 --apihost 127.0.0.1  --ton-mode 6"
-------------------------------------------------------
Error: 
No user account or wallet to mine on configured.

Has anyone ever been in this situation,thanks!

**Comment by jgonzis** [comment]
> Both1.48 & 1.49 mine_ton.bat: set "TONPOOL=stratum+ssl://xxxx:xxxx" set "TONWALLET=xxxxx.xxxxx"
> 
> ## set "EXTRAPARAMETERS=--apiport 1111 --apihost 127.0.0.1 --ton-mode 6"
> Error: No user account or wallet to mine on configured.
> 
> Has anyone ever been in this situation,thanks!

Could you share the miner log stats of 60s? with all the information to know where it could be a problem

**Comment by GYKrauss** [comment]
Using same configurations as previously working versions, my dual mining does not start TON mining, only ETH is working in 1.50

**Comment by jgonzis** [comment]
> For a build with a FHR and LHR card inside: Using same configurations as previously working versions, my dual mining does not start TON mining, only ETH is working in 1.50 A computer with only a 3060 v2 is working now ETH+TON is working well at about 48/1000.

Could you share please the miner log and the OC too?. At the moment I cannot help you without that information

**Comment by GYKrauss** [comment]
Can't replicate.... I ran it again and its working

**Comment by kindman4** [comment]
Could you share the miner log stats of 60s? with all the information to know where it could be a problem
-----------
Sure I will see log later.

**Comment by kindman4** [comment]
I add a log on EXTRAPARAMETERS,there is a log file,but no log at all,also 
-----------------------------------------------------------
Error:No user account or wallet to mine on configured.
-----------------------------------------------------------
set "EXTRAPARAMETERS=--apiport 1111 --apihost 127.0.0.1 --maxdualimpact *,*,*,*,*,*,*,* --log on"
-----------------------------------------------------------
set "TONPOOL=https://ton-pool-server-p3agi.ondigitalocean.app"
set "TONWALLET=EQCdX8XXXXXXXXXXXXXXXXXX.ton"									

set "EXTRAPARAMETERS=--apiport 1111 --apihost 127.0.0.1 --maxdualimpact *,*,*,*,*,*,*,* --log on"

+---------------------------------------------------------+
|  _       _ __  __ _                   _  _  _    ___    |
| | | ___ | |  \/  (_)_ __   ___ _ __  / || || |  ( _ )   |
| | |/ _ \| | |\/| | | '_ \ / _ \ '__| | || || |_ / _ \   |
| | | (_) | | |  | | | | | |  __/ |    | ||__   _| (_) |  |
| |_|\___/|_|_|  |_|_|_| |_|\___|_|    |_(_) |_|  \___/   |
|                                                         |
|              This software is for mining                |
|              Autolykos V2                               |
|              Ethash, Etchash                            |
|              Equihash 144/5, 192/7, 210/9               |
|              BeamHash III                               |
|              ZelHash     (EquihashR 125/4/0)            |
|              SHA256 (TON)                               |
|              Blake3 (Alephium)                          |
|              Cuck(ar)oo 29                              |
|              Cuckaroo   30 CTX                          |
|              Cuckatoo   31/32                           |
|                                                         |
|                                                         |
|             Made by Lolliedieb, April 2022              |
+---------------------------------------------------------+

**Comment by jgonzis** [comment]
> ## I add a log on EXTRAPARAMETERS,there is a log file,but no log at all,also
> ## Error:No user account or wallet to mine on configured.
> ## set "EXTRAPARAMETERS=--apiport 1111 --apihost 127.0.0.1 --maxdualimpact _,_,_,_,_,_,_,_ --log on"
> set "TONPOOL=https://ton-pool-server-p3agi.ondigitalocean.app" set "TONWALLET=EQCdX8XXXXXXXXXXXXXXXXXX.ton"
> 
> set "EXTRAPARAMETERS=--apiport 1111 --apihost 127.0.0.1 --maxdualimpact _,_,_,_,_,_,_,_ --log on"
> 
> +---------------------------------------------------------+ | _ _ __ __ _ _ _ _ ___ | | | | ___ | | / (_)_ __ ___ _ __ / || || | ( _ ) | | | |/ _ | | |/| | | '_ \ / _ \ '**| | || || |_ / _ \ | | | | (_) | | | | | | | | | **/ | | ||** _| (_) | | | |_|_**/|_|_| |_|_|_| |_|_**|_| |_(_) |_| _**/ | | | | This software is for mining | | Autolykos V2 | | Ethash, Etchash | | Equihash 144/5, 192/7, 210/9 | | BeamHash III | | ZelHash (EquihashR 125/4/0) | | SHA256 (TON) | | Blake3 (Alephium) | | Cuck(ar)oo 29 | | Cuckaroo 30 CTX | | Cuckatoo 31/32 | | | | | | Made by Lolliedieb, April 2022 | +---------------------------------------------------------+

maxdualimpact should use * to Separate. Is it used?

**Comment by kindman4** [comment]
Yes,I always use it,  It is always error like this and no log show:
Error:No user account or wallet to mine on configured.
----------
set "TONPOOL=https://ton-pool-server-p3agi.ondigitalocean.app"
set "TONWALLET=xxxxxxxxxxxxxxxxxxxxxxx.iceton"									

set "EXTRAPARAMETERS=--apiport 1111 --apihost 127.0.0.1 --maxdualimpact *,*,*,*,*,*,*,* --log on"

**Comment by kindman4** [comment]
Github disable the Shift 8 or star...

**Comment by jgonzis** [comment]
> Github disable the Shift 8 or star...

Remove .iceton  and put in EXTRAPARAMETERS --dualworker iceton
